src/SFA.DAS.EAS.Web/Orchestrators/EmployerAccountPayeOrchestrator.cs

```diff
@@ -182,6 +182,7 @@ public EmployerAccountPayeOrchestrator(IMediator mediator, ILog logger, ICookieS
             return new OrchestratorResponse<RemovePayeSchemeViewModel> { Data = model };
         }
 
+        //TODO: The message gets mutated by the method. Message needs to be made immutable
```

Comment:
Fair enough, makes sense. We originally didn't do this as it wasn't getting passed around everywhere but happy to make it immutable. Need to mention this to Michael just so he's aware of this as its a common features across projects

Reply:
Michael is aware of the immutability features in the Activities work. It's partly at his request.

Reply:
cool

Reply:
This is still marked as requiring changes